### PR TITLE
Adding Beta Banner.

### PR DESCRIPTION
**Pull request by nibivi77**
### Description
Added Banner at the top of the page.
Added rendering simple unit tests.

### CODE REVIEW

**Review by DaceyTom2 — CHANGES_REQUESTED** [review]
Looks fantastic! Small change to match with the design

File: components/PhaseBanner.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            href={props.bannerLinkHref}
+            className="text-deep-blue-dark hover:text-blue-hover"
+          >
+            <span className="mr-2 underline">{props.bannerLink}</span>
+            <FontAwesomeIcon
+              icon={icon['arrow-up-right-from-square']}
+            ></FontAwesomeIcon>
+          </a>
+        </div>
+        <Button
```
[the remark this answers is not in the document]
Comment (nibivi77) [reply components/PhaseBanner.js:29]: Good catch thanks! I didn't see there was mobile designs.

[the remark this answers is not in the document]
Comment (DaceyTom2) [reply components/PhaseBanner.js:29]: Same, I had to ask for them haha

**Review by DaceyTom2 — APPROVED** [review]
Beautiful

**Pull request merged**